test_cromwell.sh

```diff
@@ -11,51 +11,50 @@ trap "shutdown" EXIT
 
 set -e
 
-EXIT_CODE=31
+EXIT_CODE=1
 PROGNAME="$(basename $0)"
 
 usage="
-$PROGNAME [-b branch] [-r rundir] [-c config file] [-p parallelism factor]
+$PROGNAME [-b branch] [-j jar path] [-r rundir] [-c config file] [-p parallelism factor]
 
 Builds and runs specified branch of Cromwell and runs Centaur against it.
 
 Arguments:
-    -b    Branch of Cromwell to test
+    -b    Branch of Cromwell to test. Mutually exclusive with -j
+    -j    Path of a cromwell jar to use. Mutually exclusive with -b
     -r    Directory where script is run (defaults to current directory)
     -c    If supplied, the config file to pass to Cromwell
     -p    Number of simultaneous tests to run. Defaults to 3
 "
 
 INITIAL_DIR=$(pwd)
-
-PARALLELISM_FACTOR=3
-CROMWELL_BRANCH="develop"
 RUN_DIR=$(pwd)
-CONFIG_STRING=""
```

Comment:
Was this removed on purpose? Seems useful (from my "don't forget to initialise to null" days)